tests/bson/bson-int64-operation-001.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+MongoDB\BSON\Int64 operations: arithmetics
+--FILE--
+<?php
+
+$value = new MongoDB\BSON\Int64(10);
+$int64Max = new MongoDB\BSON\Int64('9223372036854775807');
+$int64Min = new MongoDB\BSON\Int64('-9223372036854775808');
+
+var_dump($value + 2);
+var_dump($value - 2);
+var_dump($value * 2);
+var_dump($value / 2);
+var_dump($value % 3);
+
+var_dump($int64Max + 1);
+var_dump($int64Min - 1);
+var_dump($int64Max * 2);
+
+// This division produces a float
+var_dump($int64Max / 2);
+
+// This division produces an int
+var_dump(($int64Max - 1) / 2);
+
+// This produces overflows
+var_dump($int64Min * -1);
+var_dump($int64Min / -1);
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+object(MongoDB\BSON\Int64)#%d (%d) {
+  ["integer"]=>
+  string(2) "12"
+}
+object(MongoDB\BSON\Int64)#%d (%d) {
+  ["integer"]=>
+  string(1) "8"
+}
+object(MongoDB\BSON\Int64)#%d (%d) {
+  ["integer"]=>
+  string(2) "20"
+}
+object(MongoDB\BSON\Int64)#%d (%d) {
+  ["integer"]=>
+  string(1) "5"
+}
+object(MongoDB\BSON\Int64)#%d (%d) {
+  ["integer"]=>
+  string(1) "1"
+}
+float(9.22%dE+18)
+float(-9.22%dE+18)
+float(1.844%dE+19)
+float(4.61%dE+18)
```

Comment:
Is PHP consistent about using "E" for scientific notation? Unrelated to PHP, I know we had an issue in BSON tests where "E" or "e" might be used on different platforms.

The [grammar definition](https://www.php.net/manual/en/language.types.float.php) in PHP.net shows `[eE]`, so I might suggest using an inline regex pattern such as `%r(e|E)%r` or `%r[eE]%r`.

This applies to anywhere else you might be expecting scientific notation in EXPECTF.

Reply:
Updated.